cpp/cmake/modules/raft_export.cmake

```diff
@@ -210,6 +210,10 @@ function(raft_export type project_name)
       file(MAKE_DIRECTORY "${scratch_dir}")
       install(DIRECTORY "${scratch_dir}" DESTINATION "${install_location}"
         COMPONENT raft_${comp})
+      if(EXISTS "${scratch_dir}/raft-${comp}-dependencies.cmake")
+        install(FILES "${scratch_dir}/raft-${comp}-dependencies.cmake" DESTINATION "${install_location}"
```

Comment:
Do we need to do something similar for the `raft-${comp}-targets.cmake` files?

Reply:
No we don't as we place the `raft-${comp}-targets.cmake` file in a location that is already installed by the `raft` component.  I also we able to remove this install command, as we now also place the `raft-${comp}-dependencies.cmake` file in the correct location.

- raft-*.cmake => installed as part of the `raft` component
- raft-nn-*.cmake => installed as part of the `raft` component
- raft-distance-*.cmake => installed as part of the `raft` component
- raft-nn-lib-*.cmake => installed as part of the `nn` component
- raft-distance-lib-*.cmake => installed as part of the `distance` component